unlock-app/src/pages/about.js

```diff
@@ -120,9 +129,9 @@ const Person = ({ name, picture, personalLink, linkedIn, bio }) => (
     <Photo src={picture} />
     <p>{bio}</p>
     <small>
-      <a href={personalLink}>Personal site</a>
+      <a href={personalLink} target="_blank">Personal site</a>
```

Comment:
✅I think this is right!

Reply:
had some eslint errors, sorry about that!